packages/pages/src/vite-plugin/build/closeBundle/manifest.ts

```diff
@@ -29,11 +29,13 @@ export const generateManifestFile = (
 
   // Add the renderPaths to the manifest. This defines the _client and _server entries.
   const renderPaths = glob.sync(
```

Comment:
Make sure that all of the paths that are output to dist/plugin/manifest.json are posix-style.

Reply:
Just checked. All of the paths there are posix-style.